### PR TITLE
chore: update database migration README.md & fix docker-compose port assignments

### PR DESCRIPTION
**Pull request by metakuni**
### Reviewers
**Functional:** 

**Readability:** 

---

## Changes
- modify:
  - update database migration `README.md`
  - fixed port assignments in `docker-compose.yml`

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -116,7 +116,7 @@ services:
         - HAPPY_TAG
     restart: "no"
     ports:
-      - "9000:8080"
+      - "9001:8080"
```
Comment (metakuni) [inline docker-compose.yml:119]: Fixed duplicate port mapping.

File: docker-compose.yml [hunk 2]
```diff
@@ -188,7 +188,8 @@ services:
       - localstack
       - database
     ports:
-      - "5000:5000"
+      # port 5000 is 'unofficially' used by Apple's Control Center! 🤬
+      - "5001:5000"
```
Comment (metakuni) [inline docker-compose.yml:192]: Avoiding an unofficial but well-known used port on macOS.

File: backend/database/README.md [hunk 3]
```diff
@@ -2,94 +2,103 @@
...
-1. Update the `Revision ID` and the `revision` (used by alembic) to include the migration count
+   This generates a file in `$REPO_ROOT/backend/database/versions/xxxxxxxxxxxx_purpose_of_migration.py`
+1. In the generated file, edit the `upgrade()` and `downgrade()` functions such that `upgrade()` contains the Alembic DDL commands to perform the migration you would like and `downgrade()` contains the commands to undo it.
+1. Rename the generated file by prepending the migration count to the filename (`xxx_purpose_of_migration.py` -> `03_xxx_purpose_of_migration.py`)
+1. In the generated file, update the `Revision ID` and the `revision` (used by Alembic) to include the migration count.
 For example `Revision ID: a8cd0dc08805` becomes `Revision ID: 18_a8cd0dc08805` and `revision = "a8cd0dc08805"` becomes `revision = "18_a8cd0dc08805"` 
 1. [Test your migration](#test-a-migration)
-1. Check that [corpora.orm](../corpora/common/corpora_orm.py) matches up with your changes.
-1. Once you've completed the changes, create a PR to get the functions reviewed. 
+1. Check that [corpora.orm.py](../corpora/common/corpora_orm.py) matches up with your changes.
```
Comment (Bento007) [inline backend/database/README.md:19]:
```suggestion
1. Check that [corpora_orm.py](../corpora/common/corpora_orm.py) matches up with your changes.
```

**Review by Bento007 — APPROVED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (MDunitz) [reply docker-compose.yml:119]: Thanks!

**Review by MDunitz — APPROVED** [review]
lgtm.... although it might be worth splitting into 2 separate prs for clarity

File: backend/Makefile [hunk 4]
```diff
@@ -40,11 +40,11 @@ db/rollback:
 
 db/new_migration:
 	# Usage: make db/new_migration MESSAGE="purpose_of_migration"
-	PYTHONPATH=.. alembic -c=./config/database.ini revision --message $(MESSAGE)
+	PYTHONPATH=.. alembic -c=./config/database.ini revision --message "$(MESSAGE)"
 
 db/new_migration_auto:
-	# Usage: make db/new_migration MESSAGE="purpose_of_migration"
-	PYTHONPATH=.. alembic -c=./config/database.ini revision --autogenerate --message $(MESSAGE)
+	# Usage: make db/new_migration_auto MESSAGE="purpose_of_migration"
```
Comment (metakuni) [inline backend/Makefile:46]: Wrong make target specified in the comment

File: backend/Makefile [hunk 5]
```diff
@@ -40,11 +40,11 @@ db/rollback:
...
+	PYTHONPATH=.. alembic -c=./config/database.ini revision --autogenerate --message "$(MESSAGE)"
```
Comment (metakuni) [inline backend/Makefile:47]: Added quotes so that snake-casing the message isn't necessary.

**Pull request merged**